yaptide/routes/celery_routes.py

```diff
@@ -16,7 +16,7 @@
                                   get_input_files, cancel_simulation)
 
 
-class SimulationRun(Resource):
+class JobsDirect(Resource):
     """Class responsible for SHIELD-HIT12A simulations running"""
```

Comment:
update name if class is responsible only for simulations in celery directly on backend